install

```diff
@@ -79,7 +79,7 @@ while true; do
             shift ;;
         -c|--clients)
             build_clients=true
```

Comment:
change build_clients to build_tests? Let's standardize the names to reflect the package names we choose.